plugins/out_stackdriver/stackdriver.c

```diff
@@ -761,21 +761,67 @@ static int get_stream(msgpack_object_map map)
     return STREAM_UNKNOWN;
 }
 
+static int make_bool_map(struct flb_stackdriver *ctx, msgpack_object *map,
+                         bool_map_t *bool_map, int map_num, 
+                         flb_sds_t to_be_removed[], int removed_num) {
+    msgpack_object_kv *kv;
+    msgpack_object *key;
+
+    char is_to_delete;
+    int i, j;
+    int ret = 0;
+    flb_sds_t removed;
+
+    for (i = 0; i < map_num; i++) {
```

Comment:
what about using ```memset(3)```  to avoid the iteration ?

Reply:
updated. I have switched the order of bool_map_t as the `TO_BE_REMAINED` is used be 1. So I switched the order to make it 0 in order to use memset.